llama-index-integrations/node_parser/relational/llama-index-node-parser-relational-dashscope/README.md

```diff
@@ -0,0 +1,42 @@
+# LlamaIndex Node_Parser-Relational Integration: Dashscope
+
+Transform your documents into nodes with ease using the Dashscope integration for LlamaIndex. This tool allows for precise control over chunk size, overlap size, and more, tailored for the Dashscope reader output format.
+
+## Quick Start
+
+Get up and running with just a few lines of code:
+
+```python
+import json
+import os
+from llama_index.node_parser.relational.dashscope import (
+    DashScopeJsonNodeParser,
+)
+from llama_index.core.ingestion import IngestionPipeline
+from llama_index.core.schema import Document
+
+# Set your Dashscope API key in the environment
+os.environ["DASHSCOPE_API_KEY"] = "your_api_key_here"
+
+documents = [
+    # Prepare your documents obtained from the Dashscope reader
```

Comment:
Is there a dashscope reader in llama-index? If not, maybe we should at least have some link here for how to use the reader or where to go to get started with an API key?

Reply:
I add the dashscope reader and index manager together. The node parse is recommended to be used with them.